apps/scraper-runner/src/main.ts

```diff
@@ -0,0 +1,79 @@
+import * as scholarsScrape from '@foodmedicine/scholars-scraper';
+import * as articleParser from '@foodmedicine/article-parser';
+import * as healthSiteScraper from '@foodmedicine/health-site-scraper';
+import * as fs from 'fs';
+import {
+  ParsedArticleParagraphStandalone,
+  ParsedArticle,
+  ParsedArticleParagraph,
+} from '@foodmedicine/interfaces';
+
+/**
+ * Get the top percentage of an array
+ * @param arr an array sorted in descending order
+ * @param percent percentage of items to be returned
+ */
+function getTopPercentage(arr: any[], percent = 5): any[] {
+  return arr.slice(0, Math.floor((arr.length * percent) / 100));
+}
+
+async function main() {
+  const healthRemedies = await healthSiteScraper.runAllScrapers();
+  const findCorrelatedParagraphsProms = healthRemedies.map(
+    async (healthRemedy) => {
+      const recommendationResults = healthRemedy.recommendations.map(
+        (recommendation) =>
+          findCorrelatedParagraphs(healthRemedy.impacted, recommendation)
+      );
+      return await Promise.all(recommendationResults);
+    }
+  );
+  await Promise.all(findCorrelatedParagraphsProms);
+}
+
+async function findCorrelatedParagraphs(
+  impacted: string,
+  recommendation: string
+) {
+  console.log(
+    `Starting to find correlation of ${recommendation} for ${impacted}`
+  );
```

Comment:
Delete?

Reply:
Can I keep these as a way of just seeing how progress is being made?

Reply:
Sure, we're just used to flagging these.

If you prefer to keep these as logs, then ignore this.  I tend to convert logs I want to keep around for a while to console.warn calls as an easy way to distinguish them, so I can do a global search for console.log calls before I submit a change and make sure I got rid of any logs I was using for temporary debugging

Reply:
Oh ok, changed. I usually use a logger, but I thought that would not be necessary for this project